src/select.coffee

```diff
@@ -64,10 +64,10 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
   placeholder:        undefined
   disabled: no
 
+  # we need to set tabindex so that div responds to key events
```

Comment:
Your revert is right. This place is wrong not because of the revert. I fixed the position that's why it was in my commit.